<!DOCTYPE html> <html lang="en-US"> <head> <title>On the rapid calculation of binding affinities for antigen and antibody design and affinity maturation simulations</title> <meta charset="UTF-8"> <meta name="author" content="Simone Conti"> <meta name="viewport" content="width=device-width, initial-scale=1.0"> <link rel="canonical" href="https://3dz2.com/publications/conti2022rapid/" /> <!-- Twitter --> <meta name="twitter:card" content="summary"> <meta name="twitter:url" content="https://3dz2.com/publications/conti2022rapid/"> <meta name="twitter:title" content="On the rapid calculation of binding affinities for antigen and antibody design and affinity maturation simulations"> <meta name="twitter:site" content="@simonecnt"> <meta name="twitter:creator" content="@simonecnt"> <!-- Open Graph --> <meta property="og:title" content="On the rapid calculation of binding affinities for antigen and antibody design and affinity maturation simulations" /> <meta property="og:type" content="article" /> <meta property="og:url" content="https://3dz2.com/publications/conti2022rapid/" /> <meta property="og:locale" content="en_US" /> <link rel="icon" type="image/svg+xml" href="/img/favicon.svg"> <meta name="google-site-verification" content="riBpdQWal9w_jxf97W3pf6jUQs7jkzT3U2UOi9MZuPY" /> <link rel="stylesheet" href="/css/w3.css"> <link rel="stylesheet" href="/css/main.css" type="text/css" /> <link rel="stylesheet" href="/css/pygments-default.css" type="text/css" /> <script src="https://kit.fontawesome.com/cf75892fab.js" crossorigin="anonymous"></script> <link href='https://fonts.googleapis.com/css?family=Open+Sans:300,600' rel='stylesheet' type='text/css'> <style>body{font-family: 'Open Sans', sans-serif;}</style> <!-- Global site tag (gtag.js) - Google Analytics --> <script async src="https://www.googletagmanager.com/gtag/js?id=G-SS8LGHNGNZ"></script> <script> window.dataLayer = window.dataLayer || []; function gtag(){dataLayer.push(arguments);} gtag('js', new Date()); gtag('config', 'G-SS8LGHNGNZ'); </script> </head> <body> <header> <div id="title"> <h1>3dz2.com</h1> <h2></h2> </div> <img src="/img/banners/stampa.png" alt="image banner" /> </header> <main> <article> <h1>On the rapid calculation of binding affinities for antigen and antibody design and affinity maturation simulations</h1> <hr /> <p> Simone Conti, Edmond Y. Lau, and Victor Ovchinnikov </p> <p> <em>Antibodies</em>, 2022, 11(3), pp 1-19.<br /> <a href="http://dx.doi.org/10.3390/antib11030051" target="_blank">DOI:10.3390/antib11030051</a> | <a href="https://www.researchgate.net/publication/362475292" target="_blank" >Find on RG</a> </p> <figure class="graphabstract" ><img src="/img/papers/conti2022rapid.jpg" alt="On the rapid calculation of binding affinities for antigen and antibody design and affinity maturation simulations" /></figure> <h3>Abstract:</h3> <p>The accurate and efficient calculation of protein-protein binding affinities is an essential component in antibody and antigen design and optimization, and in computer modeling of antibody affinity maturation. Such calculations remain challenging despite advances in computer hardware and algorithms, primarily because proteins are flexible molecules, and thus, require explicit or implicit incorporation of multiple conformational states into the computational procedure. The astronomical size of the amino acid sequence space further compounds the challenge by requiring predictions to be computed within a short time so that many sequence variants can be tested. In this study, we compare three classes of methods for antibody/antigen (Ab/Ag) binding affinity calculations: (i) a method that relies on the physical separation of the Ab/Ag complex in equilibrium molecular dynamics (MD) simulations, (ii) a collection of 18 scoring functions that act on an ensemble of structures created using homology modeling software, and (iii) methods based on the molecular mechanics-generalized Born surface area (MM-GBSA) energy decomposition, in which the individual contributions of the energy terms are scaled to optimize agreement with the experiment. When applied to a set of 49 antibody mutations in two Ab/HIV gp120 complexes, all of the methods are found to have modest accuracy, with the highest Pearson correlations reaching about 0.6. In particular, the most computationally intensive method, i.e., MD simulation, did not outperform several scoring functions. The optimized energy decomposition methods provided marginally higher accuracy, but at the expense of requiring experimental data for parametrization. Within each method class, we examined the effect of the number of independent computational replicates, i.e., modeled structures or reinitialized MD simulations, on the prediction accuracy. We suggest using about ten modeled structures for scoring methods, and about five simulation replicates for MD simulations as a rule of thumb for obtaining reasonable convergence. We anticipate that our study will be a useful resource for practitioners working to incorporate binding affinity calculations within their protein design and optimization process.</p> </article> </main> <aside> <!-- Contents --> <div class="sideblock"> <nav> <h3>Contents</h3> <hr /> <ul> <li><i class="fas fa-home fa-fw"></i> <a href="/">Home</a> </li> <li><i class="fas fa-user-circle fa-fw"></i> <a href="/cv/">CV</a> </li> <li><i class="fas fa-flask fa-fw"></i> <a href="/research/">Research Projects</a> </li> <li><i class="fas fa-book fa-fw"></i> <a href="/publications/">Publications</a> </li> <li><i class="fas fa-code fa-fw"></i> <a href="/programming/">C Programming</a> </li> <li><i class="fas fa-gem fa-fw"></i> <a href="/minerals/">Minerals</a> </li> <li><i class="fas fa-star fa-fw"></i> <a href="/mineralapp/">MineralApp</a> </li> <li><i class="fas fa-paper-plane fa-fw"></i> <a href="/stuff/">Stuff</a> </li> <li><i class="fas fa-envelope fa-fw"></i> <a href="/contact/">Contact</a> </li> </ul> </nav> </div> </aside> <footer> <p>Copyright &copy; 2015-2022 Simone Conti. All rights reserved.</p> </footer> </body> </html>
